index.js

```diff
@@ -46,6 +46,17 @@ module.exports = function(options, cb) {
           return next(new Error('Logging dirver is awslogs. Can not stream logs unless logging driver is awslogs'));
         }
 
+        if (result.taskDefinition.networkMode === 'awsvpc') {
+          if (options.subnets === undefined || options.securityGroups === undefined) {
+            return next(new Error('Task definition networkMode is awsvpc, this requires you to specify subnets and security-groups.'));
```

Comment:
`securityGroups` is not a required property. The task could be started without one.